### PR TITLE
Optional features: add a section on custom branding

### PR DESCRIPTION
**Pull request by bahmady**
### Context
Users need to know how they can get custom branding - i.e. what they need to submit to the GOV.UK Pay team

### Changes proposed in this pull request
Adds a section on custom branding (for users) in "Optional features" section

### Guidance to review
This will need reviewing by SMEs before it can be merged

Viewable as if _in situ_ at https://govukpay-docs-optional-features-test.cloudapps.digital/optional_features/#optional-features

### CODE REVIEW

**Comment by bahmady** [comment]
Hi @jonheslop would you be able to check this PR out to see if the instructions on what people need to send the GOV.UK Pay team to get custom branding set up make sense?

File: source/documentation/custom_branding.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+GOV.UK Pay supports custom branding on payment pages.
+
+You can [contact the GOV.UK Pay](/support_contact_and_more_information) to
+request customisation for the following features:
+
+* the logo displayed in the left-hand side of the top banner
+* the background and border colour of the top banner
+* your payment confirmation email from GOV.UK Notify
```
Comment (bahmady) [inline source/documentation/custom_branding.md]: @katiebates @karlchillmaid @mpwoods is there somewhere we can link to about this guidance?

Comment (karlchillmaid) [reply source/documentation/custom_branding.md]: To request custom branding for Notify email templates you need to be signed in to your account. In _Settings_ there's a field called _Email branding_ where you can choose the type of customisation you want from a list of available options. This sends a request to the Notify team, who contact the user directly to request logos etc.
At the moment there isn't any detailed information available when signed out of Notify.
Branding is mentioned on the _Features_ page, but it's probably not informative enough to link to from here.

Comment (bahmady) [reply source/documentation/custom_branding.md]: OK, thanks @karlchillmaid. Unless we have some further guidance I can link to I'm a little bit hesitant to include that line in this documentation unless we can be assured that people will already have enough context about the subject by this point.

**Comment by bahmady** [comment]
@katiebates https://github.com/alphagov/pay-tech-docs/pull/158#event-1858257677 made all the "optional features" on a single page: was this intentional? Fine with it but just wanted to check as I previously had them on separate pages.

**Pull request merged**